api/kessel/inventory/v1beta1/reporter_data.proto

```diff
@@ -1,25 +1,25 @@
 syntax = "proto3";
 
-package api.kessel.inventory.v1beta1;
+package kessel.inventory.v1beta1;
 
 import "google/api/field_behavior.proto";
 import "google/protobuf/timestamp.proto";
 
-option go_package = "api/kessel/inventory/v1beta1;v1beta1";
+option go_package = "github.com/project-kessel/inventory-api/api/kessel/inventory/v1beta1";
 option java_multiple_files = true;
 option java_package = "api.kessel.inventory.v1beta1";
 
 message ReporterData {
 
-  enum Reporter_typeEnum {
-    Reporter_typeEnum_ACM = 0;
-    Reporter_typeEnum_HBI = 1;
-    Reporter_typeEnum_OCM = 2;
-    Reporter_typeEnum_OTHER = 3;
-    Reporter_typeEnum_UNKNOWN = 4;
+  enum ReporterTypeEnum {
+    REPORTER_TYPE_ENUM_ACM_UNSPECIFIED = 0;
+    REPORTER_TYPE_ENUM_HBI = 1;
+    REPORTER_TYPE_ENUM_OCM = 2;
+    REPORTER_TYPE_ENUM_OTHER = 3;
+    REPORTER_TYPE_ENUM_UNKNOWN_UNSPECIFIED = 4;
```

Comment:
```suggestion
    REPORTER_TYPE_ENUM_UNKNOWN = 0;
    REPORTER_TYPE_ENUM_ACM = 1;
    REPORTER_TYPE_ENUM_HBI = 2;
    REPORTER_TYPE_ENUM_OCM = 3;
    REPORTER_TYPE_ENUM_OTHER = 4;
```

Don't overload ACM and unknown.  I've updated to "UNKNOWN" based on previous state.  This is not consistent with other enums which use "UNKNOWN_UNSPECIFIED".  I'd prefer we pick one word/phrase and stick to it for all enums, I don't have a preference on what specifically.

Reply:
Buf lint suggest for `UNKNOWN`  to append a suffix for UNSPECIFIED 

eg:` kessel/inventory/v1beta1/reporter_data.proto:15:5:Enum zero value name "Reporter_typeEnum_ACM" should be suffixed with "_UNSPECIFIED".`

we can `REPORTER_TYPE_ENUM_UNKNOWN` to `REPORTER_TYPE_ENUM_UNSPECIFIED`

Reply:
Sure, using `UNSPECIFIED` works.  So a few replaces?  `/UNKNOWN/UNSPECIFIED/` and `/UNKNOWN_UNSPECIFIED/UNSPECIFIED/`?